client/state/selectors/get-site-default-post-format.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getRawSite, getSiteOption } from 'state/sites/selectors';
+import { getSiteSettings } from 'state/site-settings/selectors';
+
+/**
+ * Returns the default post format of a site.
+ * Returns null if the site is unknown.
+ *
+ * @param  {Object}    state   Global state tree
+ * @param  {Number}    siteId  The ID of the site we're querying
+ * @return {?String}           The default post format of that site
+ */
+export default function getSiteDefaultPostFormat( state, siteId ) {
+	const site = getRawSite( state, siteId );
```

Comment:
Why do we need to call `getRawSite` at all? Wouldn't `getSiteOption` and `getSiteSettings` safely fall back to an empty value if the site data isn't available?